supervisor/resolution/module.py

```diff
@@ -46,6 +46,12 @@ def __init__(self, coresys: CoreSys):
         self._unsupported: list[UnsupportedReason] = []
         self._unhealthy: list[UnhealthyReason] = []
 
+    async def load_modules(self):
+        """Load resolution evaluation, check and fixup modules."""
+        await self._evaluate.load_modules()
+        await self._check.load_modules()
+        await self._fixup.load_modules()
```

Comment:
I think we should change each of these 3 `load_modules` modules to not be `async` and run all of them in the executor from here. Rather then having each one do its own loading in a separate executor task.

I know since each of them is setting class data this is normally something we would avoid doing in the executor. But I believe it should be safe in this case given the point at which this code is called. There is nothing else running that could be looking at resolution center data until this completes since we're still in bootstrap.

If you want to be extra safe can make those methods raise if called a second time to ensure they cannot be used outside of bootstrap accidentally and create race conditions.